bsd/backends.py

```diff
@@ -0,0 +1,50 @@
+from bs4 import BeautifulSoup
+from django.contrib.auth.models import User
+from StringIO import StringIO
+from xml.etree.ElementTree import ElementTree
+from .api import Bsd
+
+
+# TODO: what is best way to do this?
+api = Bsd().api
+
+class BSDAuthenticationBackend:
```

Comment:
2 blank lines here, according to PEP8.  https://www.python.org/dev/peps/pep-0008/#id21